scale/job/test/execution/configuration/test_configurators.py

```diff
@@ -452,13 +504,32 @@ def test_configure_scheduled_job_ingest(self, mock_msg_mgr):
         scan = ingest_test_utils.create_scan()
         ingest_job_type = Ingest.objects.get_ingest_job_type()
         ingest = ingest_test_utils.create_ingest(scan=scan, workspace=workspace, new_workspace=new_workspace)
-        Ingest.objects.start_ingest_tasks([ingest], scan_id=scan.id)
 
+        # start ingest_tasks (can't call method due to using command messages)
+        when = ingest.transfer_ended if ingest.transfer_ended else now()
```

Comment:
more code duplication; alternatively, this code could be moved into a test utility method

Reply:
Message executed!